docs/src/main/asciidoc/openshift-s2i-guide.adoc

```diff
@@ -79,6 +79,42 @@ oc patch bc/quarkus-quickstart-native -p '{"spec":{"resources":{"limits":{"cpu":
 oc start-build quarkus-quickstart-native
 ----
 
+== Building a minimal runtime container
+
+As an alternative to creating an application from the S2I build process one can use chained builds to produce a runner image that is minimal in size and does not contain all the dependencies required to build the application.
```

Comment:
The minimal runtime is not only an extension. Please see the comment about the TIP. It's more like an alternative for users that want to have the minimal image as the runtime instead of the S2I image. It does, however, depend on the BC from the above section.